### PR TITLE
fix(admin): correct the count field of plugin-metadata/global-rule

### PR DESCRIPTION
**Pull request by mangoGoForward**
### What this PR does / why we need it:
Correct the count field of plugin-metadata. please see [comment](https://github.com/apache/apisix/issues/6105#issuecomment-1012855448)

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [x] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

**Review by spacewander — COMMENTED** [review]
Could you also solve the `global_rules` in this PR?

**Comment by mangoGoForward** [comment]
> Could you also solve the `global_rules` in this PR?

Ok, I will commit later.

**Comment by mangoGoForward** [comment]
> Could you also solve the `global_rules` in this PR?

Already done. please see 7a2ce71.

File: t/admin/global-rules2.t [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    if (!$block->no_error_log && !$block->error_log) {
+        $block->set_value("no_error_log", "[error]\n[alert]");
+    }
+});
+
+run_tests;
+
+__DATA__
+
+=== TEST 1: list empty resources
```
Comment (tokers) [inline t/admin/global-rules2.t:41]: Can we add some test cases for non-empty resources?

Comment (mangoGoForward) [reply t/admin/global-rules2.t:41]: Done. please see b94a86c.

File: t/admin/global-rules2.t [hunk 2]
```diff
@@ -61,3 +61,91 @@ __DATA__
...
+
+            res = json.decode(res)
+            res.node.value.create_time = nil
+            res.node.value.update_time = nil
+            ngx.say(json.encode(res))
+        }
+    }
+--- response_body
+{"action":"set","node":{"key":"/apisix/global_rules/1","value":{"id":"1","plugins":{"proxy-rewrite":{"uri":"/"}}}}}
+--- request
```
Comment (spacewander) [inline t/admin/global-rules2.t:98]: There is no need to repeat the two sections below?

Comment (mangoGoForward) [reply t/admin/global-rules2.t:98]: I want to add a test cases for non-empty resources like `TEST 3`, so put a new global rules.

Comment (spacewander) [reply t/admin/global-rules2.t:98]: But you already set them (--- request,  --- no_error_log) in https://github.com/apache/apisix/pull/6155/files#diff-150a3eabdca95c99b2f057a9981111fd3dc0f7c830999549b60ce791265319c3R28

Comment (mangoGoForward) [reply t/admin/global-rules2.t]: Got it, I have removed request and error log phrase.

**Review by spacewander — APPROVED** [review]

**Review by starsz — APPROVED** [review]

**Review by tzssangglass — APPROVED** [review]
